tests/acceptance/bootstrap/CollaborationContext.php

```diff
@@ -228,4 +246,53 @@ public function userTriesToCreateAFileInsideDeletedFolderUsingWopiEndpoint(strin
 			)
 		);
 	}
+
+	/**
+	 * @Given user :user has sent POST request on app endpoint:
+	 *
+	 * @param string $user
+	 * @param TableNode $items
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function userHasSentPostRequestOnAppEndpoint(string $user, TableNode $items): void {
+		$rows = $items->getRowsHash();
+		$appResponse = CollaborationHelper::sendPOSTRequestToAppOpen(
+			$this->spacesContext->getFileId($user, $rows['space'], $rows['resource']),
+			$rows['suites'],
+			$this->featureContext->getActualUsername($user),
+			$this->featureContext->getPasswordForUser($user),
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef()
+		);
+		$this->featureContext->theHTTPStatusCodeShouldBe(200, '', $appResponse);
+		$this->storeLastAppEndpointResponse($appResponse->getBody()->getContents());
+	}
+
+	/**
+	 * @When user :user tries to get the file information of file using wopi endpoint
+	 * @When user :user gets the file information of file using wopi endpoint
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function userTriesToCheckTheInformationOfDeletedFileUsingWopiEndpoint(string $user):void {
```

Comment:
```suggestion
	public function userGetsTheFileInformationUsingWopiEndpoint(string $user):void {
```
or something like this